pkg/controller/provider/container/vsphere/collector.go

```diff
@@ -495,11 +497,11 @@ func (r *Collector) watch() (list []*libmodel.Watch) {
 
 //
 // Build the client.
-func (r *Collector) connect(ctx context.Context) error {
+func (r *Collector) connect(ctx context.Context) (int, error) {
```

Comment:
If the new int had a name it would be clear what it is even without reading the body of the function. Maybe httpStatus or something like that?